### DISCUSSION TITLE
Companies & Websites Using Analog

### DISCUSSION THREAD

**Discussion by brandonroberts** [analogjs/analog#1232] [General]
If you use Analog for your website, company, or organization, please share the URL and/or GitHub repo here!

Robin Goetz - [Blog](https://goetzrobin.github.io/) [GitHub](https://github.com/goetzrobin/goetzrobin.github.io)
Matthieu Riegler aka. Jean Mèche - [Blog](https://riegler.fr/) [GitHub](https://github.com/JeanMeche/riegler.fr)
Luis Castro - [Blog](https://mrrobot.dev/) [GitHub](https://github.com/luishcastroc/blog)
Chris Perko - [Blog](https://perko.dev/) [GitHub](https://github.com/BaronVonPerko/perko-dev)
Preston Lamb - [Blog](https://www.prestonlamb.com/)
Just Angular - [Blog](https://justangular.com/) [GitHub](https://github.com/eneajaho/just-angular)
Built With Analog - [Site](https://builtwithanalog.dev)
Spartan - [Site](https://spartan.ng) [GitHub](https://github.com/goetzrobin/spartan)
Angular Hub - [Site](https://angular-hub.com) [GitHub](https://github.com/angular-sanctuary/angular-hub)

**Comment by DerHerrGammler** [comment]
Hey,

as I already shared in the discord showcase channel, my portfolio homepage is analog.
Atm very basic but will get more pages over the time:

https://herold.solutions

By the way, its a really great DX working with analog ♥️

**Comment by dalenguyen** [comment]
Built PDF open source with Analog. The speed of the development is extremely fast :D 

https://pdfun.xyz
https://github.com/dalenguyen/pdfun

Also, my portfolio is also built with Analog: https://dalenguyen.me :)

**Comment by RiddleTime** [comment]
Build for an open source project with Tailwind.
https://race.elementfuture.com/news
https://github.com/RiddleTime/Race-Element (pages branch)

**Comment by leblancmeneses** [comment]
Directory of Companies in Texas currently. 
https://www.improvingstartups.com/startupwiki/

**Comment by renatoaraujoc** [comment]
My company located at Brazil, Rio de Janeiro:

**Portal RC Ambiental** - Sells subscription based access to brazilian legislation to the biggest law firms of our country, industries and even have international clients.

URL: https://www.rcambiental.com.br
